css-positioning/fixed.css

```diff
@@ -0,0 +1,10 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  left: 200px;
```

Comment:
Set this to 0, and add `right: 0` as well, and it will center it on the screen for you